runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/WindmillServerStub.java

```diff
@@ -91,6 +91,11 @@ public abstract GetWorkStream getWorkStream(
   /** Returns a stream allowing individual WorkItemCommitRequests to be streamed to Windmill. */
   public abstract CommitWorkStream commitWorkStream();
 
+  /** Returns the amount of time the server has been throttled and resets the time to 0. */
+  public long getAndResetThrottleTime() {
```

Comment:
Make this one abstract and implement in both subtypes.

Reply:
Done